Ui/Component/Listing/Columns/CrossBorderClassActions.php

```diff
@@ -67,4 +85,16 @@ public function prepareDataSource(array $dataSource)
 
         return $dataSource;
     }
+
+    /**
+     * @param int $type_id
+     *
+     * @return null|string
+     * @throws \Magento\Framework\Exception\LocalizedException
+     */
+    protected function fetchCrossBorderTypeValue(int $type_id)
+    {
+        $type_object = $this->crossBorderTypeRepository->getById($type_id);
```

Comment:
No reason to assign the `getById` result to memory, chain the `getType` call from this

Reply:
Done.